addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1390,6 +1391,368 @@ describe('Paste', () => {
                     contentAfter: '<ul><li>123</li><li>456[]abc</li><li>def</li><li>ghi</li></ul>',
                 });
             });
+            it('should paste a nested list into another list', async () => {
+                await testEditor(BasicEditor, {
+                    contentBefore: '<ol><li>Alpha</li><li>Beta</li><li>[]</li></ol>',
```

Comment:
Without content, your `li` won't be visible and the cursor can't actually be placed there.
```suggestion
                    contentBefore: '<ol><li>Alpha</li><li>Beta</li><li><br>[]</li></ol>',
```